docs/known-issues.md

```diff
@@ -8,9 +8,12 @@ Known issue = any major user-facing bug or lack of user-facing feature that:
+| IL2CPP does not currently work on Mac as it crashes upon starting the game. This is due to a bug in Unity that does not initialize a system correctly, if one of the injected `IComponentData` contains a `long`.                                                                                                            | 2018/08/31 | 1076596      | Use the Mono compiler.                                                | Yes    |
+| `bool` is not a [blittable type](https://docs.microsoft.com/en-us/dotnet/framework/interop/blittable-and-non-blittable-types), so will cause an error if used to store component data.                                                                                                                                       | 2018/09/17 | None         | Use `BlittableBool` instead.                                          | No     |
+| Accessing component data from an entity query will segfault, therefore requests where `ResultType` is `SnapshotResultType` will be dropped before sending.                                                                                                                                                                   | 2018/09/19 | None         | Don't send entity queries where `ResultType` is `SnapshotResultType`. | No     |
+| [Writer [Require] fields](content/gameobject/reading-and-writing-component-data.md#how-to-update-component-properties): When you use `Writer.Send` to update a component's properties, it causes the SpatialOS GDK to update all properties of the component, even if you have set it to only update some of the properties. | 2018/09/25 | None         | None                                                                  | No     |
+| [Reader [Require] fields](content/gameobject/reading-and-writing-component-data.md#how-to-react-to-component-property-changes): The `ISpatialComponentUpdate update` argument of `Reader.ComponentUpdated` may indicate that a component property has changed even when it has not.                                          | 2018/09/25 | None         | Verify manually whether a component property has actually changed.    | No     |
+| [Reader [Require] fields](content/gameobject/reading-and-writing-component-data.md#how-to-react-to-component-property-changes): `Reader.<component property name>Updated` may be invoked even if `<component property name>` did not truly change.                                                                           | 2018/09/25 | None         | Verify manually whether a component property has actually changed.    | No     |
```

Comment:
truly -> really ... or just say "did not change".

Reply:
Done!